scripts/core/interactive-article-actions-panel/actions/publish-tab.tsx

```diff
@@ -53,9 +54,9 @@ export class PublishTab extends React.PureComponent<IProps, IState> {
             selectedDestination: getInitialDestination([this.props.item], false),
             publishingDateOptions: getInitialPublishingDateOptions([props.item]),
             publishingTarget: {
-                target_subscribers: [],
-                target_regions: [],
-                target_types: [],
+                target_subscribers: this.props.item.target_subscribers,
```

Comment:
you need to consider that `target_subscribers` and others might be `null` and use initial value in that case